client-side/src/stories/verticalTabs/verticalTabss.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import './verticalTabs.scss'
+const  VerticalTabs=({ labels=['tab1','tab2'],elements=[<p>tab 1</p>, <pp>tab 2</pp>]})=> {
```

Comment:
in this case you shouldn't add defaultValues because you marked the both props as isRequired,
but any way, if you need it for some reason please pay attention that:
default values shouldn't be hard-code it could be an empty array's